src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -432,6 +432,18 @@ export default class SourceFileLinter {
 				messageDetails: deprecationInfo.messageDetails,
 			});
 		}
+
+		if (this.isSymbolOfPseudoType(symbol)) {
+			this.#reporter.addMessage({
+				node: moduleSpecifierNode,
+				severity: LintMessageSeverity.Error,
+				ruleId: "ui5-linter-no-deprecated-api",
```

Comment:
As this is different from deprecated API usage, I would suggest to use a different ruleId.

My first thought would be
```suggestion
				ruleId: "ui5-linter-no-pseudo-modules",
```

Reply:
Done